main/resource-access.bs

```diff
@@ -67,13 +69,35 @@ if it needs to access resources requiring authentication (see [[#webid]]).
 When it receives a response with a `403` or `404` status code,
 it MAY repeat the request with different credentials.
 
+A Solid client MUST use the `Content-Type` request header in `PUT`, `POST` and `PATCH` requests. [[Source](https://github.com/solid/specification/issues/70#issuecomment-547924171)]
```

Comment:
Ref to RFC as to _how_ it should be used?

Reply:
Ok https://github.com/solid/specification/commit/d0b199ae46a642100dcecaba264084defe5dcae8 . Those methods will probably have their own sections going in more detail.